pkg/common/parameters.go

```diff
@@ -170,6 +174,21 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 			if paramAvailabilityClass == ParameterRegionalHardFailoverClass {
 				p.ForceAttach = true
 			}
+		case ParameterKeyEnableConfidentialCompute:
+			paramEnableConfidentialCompute, err := ConvertStringToBool(v)
+			if err != nil {
+				return p, fmt.Errorf("parameters contain invalid value for EnableConfidentialCompute parameter: %w", err)
```

Comment:
Since customers are seeing this message, it may make sense to call it enable-confidential-storage? Otherwise it is confusing.